app/types/person.ts

```diff
@@ -0,0 +1,23 @@
+import { LookupTableOption } from "./relationships";
+
+export type Person = {
+  crash_pk: number;
+  drvr_city_name: string;
+  drvr_ethncty: LookupTableOption;
+  prsn_ethnicity_id: number;
+  drvr_zip: string;
+  gndr: LookupTableOption;
+  prsn_gndr_id: number;
```

Comment:
i am not sure if i should add a comment to specify if the key is a foreignkey or leave it as an exercise to the reader?

Reply:
i think it's obvious enough 👍